android/app/src/main/java/app/organicmaps/util/Config.java

```diff
@@ -36,6 +36,8 @@ public final class Config
   private static final String KEY_DONATE_URL = "DonateUrl";
   private static final String KEY_PREF_SEARCH_HISTORY = "SearchHistoryEnabled";
 
+  public static final String KEY_PREF_LAST_SELECTED_TAB = "LastSelectedTab";
```

Comment:
`LastSearchTab` is more obvious

Reply:
Thanks for the suggestion, @pastk.
Assignment and usages updated to `LastSearchTab` and `KEY_PREF_LAST_SEARCHED_TAB` respectively.